app/models/article.rb

```diff
@@ -1,3 +1,22 @@
 class Article < ActiveRecord::Base
+  default_scope -> { order('created_at DESC') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>`default_scope` expects a block as its sole argument.